code/network/multimsgs.cpp

```diff
@@ -8011,14 +8011,16 @@ static constexpr size_t animation_forced_bit = 1 << 1;
 static constexpr size_t animation_instant_bit = 1 << 2;
 static constexpr size_t animation_pause_bit = 1 << 3;
 
-void send_animation_triggered_packet(int animationId, int pmi, const animation::ModelAnimationDirection& direction, bool force, bool instant, bool pause, const int* /*time*/) {
+void send_animation_triggered_packet(unsigned int animationId, object* parent_object, ushort special_mode, const animation::ModelAnimationDirection& direction, bool force, bool instant, bool pause, const int* /*time*/) {
 	int packet_size;
+	ushort netsig_to_send = special_mode == 0 && parent_object != nullptr ? parent_object->net_signature : (ushort)0;
 	ubyte data[MAX_PACKET_SIZE];
 
 	BUILD_HEADER(ANIMATION_TRIGGERED);
 
 	ADD_INT(animationId);
-	ADD_INT(pmi);
+	ADD_SHORT(netsig_to_send);
+	ADD_SHORT(special_mode); // Currently empty. Reserved to find special pmi's for non-object animations
```

Comment:
With these packet changes we have to bump the multi version again.  I'll probably have more comments later.